native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -70,6 +127,14 @@ class BpkTextInput extends Component {
     return this.props.value || this.state.isFocused ? 0 : 1;
   }
 
+  animatedValues: {
```

Comment:
It's not clear what the purpose of this is...

Reply:
Like many Flow related changes, the purpose was to make Flow stop complaining about perfectly good code 😆 / 😭

Reply:
The way we have done this before is placing the following before the constructor:
```
color: mixed;
labelPosition: mixed;
```

Reply:
See the example in my pr https://github.com/Skyscanner/backpack/pull/523/files#diff-6e9cfc1ddf904e496cf0a6dc816e151cR56

Reply:
I've changed it to your way and that's sorted it - I think it's because I was erroneously instantiating the variable, which caused Flow to want it to be placed below other things.

Reply:
Actually, now Flow is complaining about something else because we're using `this.animatedValues.color` in `componentDidUpdate` and it thinks it could be null, even though we're instantiating it in the constructor so it's not possible.